source/manuals/programming-languages/nodejs/style-guide.html.md.erb

```diff
@@ -0,0 +1,122 @@
+---
+title: GDS Node.js Style Guide
+---
+
+# <%= current_page.data.title %>
+
+## General style
+
+As with all JavaScript at GDS, for Node.js applications we use the [standard] style.
+
+## Use strict mode
+
+[Strict mode] should be enabled for Node.js modules, as in the browser this prevents polluting the global scope (in
+Node.js this would pollute the `global` object, rather than the `window` object).
+
+## ES2015+
+
+Because with Node.js you control the version of the interpreter you're code is running on,
+you have access to many the latest features of JavaScript, such as:
+
+- [let] and [const]
+- [Arrow functions]
+- [Array.prototype] methods like `Array.map()`, `Array.find()` and `Array.values()`
+- The [spread operator]
+
+However use caution when using cutting edge JavaScript features and be sure to read the [documentation].
```

Comment:
Maybe Stage - 3 candidates are most safe? But nothing lower? 

https://babeljs.io/docs/plugins/preset-stage-3/

If we wanted to be super cautious maybe even stage 3 is too early.

Reply:
I'd propose we avoid even stage-3 candidates and 'track v8'.

So we only use the ES6 features that are available in our current LTS Node version's V8.

This means avoiding transpiling entirely.

See https://zeit.co/blog/next#data-fetching-is-up-to-the-developer

"Our stance of what features we transpile (like async/await) can be summed up as: we target V8's capabilities. Since our goal is code-sharing between server and client, this gives us great performance when executing the code on Node and developing on Chrome or Brave, for example."